backend/config/index.ts

```diff
@@ -112,9 +113,13 @@ const config: Configuration = {
   mongodb_url:
     process.env.MONGODB_URL || "mongodb://127.0.0.1:27017/dev-aides-jeunes",
   sentry: {
-    dsn: process.env.SENTRY_BACKEND_URL,
-    authToken: process.env.SENTRY_AUTH_TOKEN,
-    project: process.env.SENTRY_PROJECT,
+    dsn: process.env?.SENTRY_BACKEND_DSN,
+    authToken: process.env?.SENTRY_BACKEND_AUTH_TOKEN,
+    project: process.env?.SENTRY_BACKEND_PROJECT,
+    integrations: [nodeProfilingIntegration()],
+    tracesSampleRate: 1.0,
+    profilesSampleRate: 1.0,
+    debug: !process.env?.SENTRY_BACKEND_DSN,
```

Comment:
Ce serait pas mieux de le mettre à `true` uniquement sur les environnements hors production ?
`"It's generally not recommended to turn it on in production, though turning debug mode on will not cause any safety concerns."`

Reply:
J'ai considéré que si on ne veux pas envoyer de log d'erreur de sentry, on ne met pas de DNS et on active le mode debug.
Cependant c'est plus clair à la lecture de mettre si différent de production. Par contre, cela signifie qu'on passe en mode debug sur la preprod.

Reply:
J'ai donc modifié en spécifiant le mode debug uniquement dans un context development.

Reply:
Merci, oui c'est plus clair à la lecture comme ça